### DISCUSSION TITLE
采用CUTI索引代替下面的其他高校列表？

### DISCUSSION THREAD

**Discussion by LaoshuBaby** [whutug/whu-thesis#69] [General]
因为如果是手工收集其他学校的同类模板的列表的话，难免有力不从心之事？或者不够全面之类

因此如果是出于这种目的，可以考虑https://github.com/LaoshuBaby/china-university-thesis-index

（啊，如果是仅仅列出有对whu-thesis有参考作用的友校模板的话，那就当我从来没提出过这个建议吧ovo）

**Comment by tanukihee** [comment]
确实只是列出了 whu-thesis 所参考的模板，没有全部收集列出的想法

readme （可能以后）会有更新，但我太懒了（逃

**Comment by LaoshuBaby** [comment]
理解理解，所以如果有见到新的模板，欢迎来CUTI列表提交？（因为看似乎还没人做过类似的索引）




------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;; 
Date: 2021年5月28日(星期五) 中午1:32
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: Re: [whutug/whu-thesis] 采用CUTI索引代替下面的其他高校列表？ (#69)





 
确实只是列出了 whu-thesis 所参考的模板，没有全部收集列出的想法
 
readme （可能以后）会有更新，但我太懒了（逃

**Reply by zepinglee** [reply]
> 看似乎还没人做过类似的索引

https://ctan.org/pkg/texproposal
https://github.com/ustctug/awesome-latex-thesis
https://wenda.latexstudio.net/article-5038.html

感觉没必要搞个 index，一般用户只需找自己学校就可以了
